packages/flutter_tools/lib/src/build_runner/devfs_web.dart

```diff
@@ -376,7 +376,8 @@ class WebDevFS implements DevFS {
   Set<String> get assetPathsToEvict => const <String>{};
 
   @override
-  Uri get baseUri => null;
+  Uri get baseUri => _baseUri;
```

Comment:
Could you add a simple unit test for the run_hot tests and the resident web runner tests?

Reply:
Done.